### PR TITLE
change(web): rework of KMW sample pages; offers differing site-layout examples 🧩

### PR DESCRIPTION
**Pull request by jahorton**
This comes, in part, out of some of our internal discussion on this Community site thread: https://community.software.sil.org/t/is-it-possible-to-make-a-webpage-with-a-local-keyboard/7504/18

The sample pages needed at least _some_ updating anyway, as the build product paths have been changed during ES-module work... and since we'd literally just been talking about some of our guides' insufficiencies, this seemed like an actionable step I could take toward improvement.  After all, the `init` function and its path-related options aren't necessarily the most clear, even with the existing guides and prior version of the samples.

Note that all samples now use 'release' versions of web, as that's what we'd expect site-developers to utilize.  I made sure to preserve linkage for KMW sourcemapping for the pages, so it's still quite possible to debug any issues there.

 @keymanapp-test-bot skip

### CODE REVIEW

File: web/src/samples/complex/root/pages/index.html [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+        // Loads keyboards from the base sample folder
+        keyman.addKeyboards({
+          id:'us',
+          name:'English',
+          languages: {
+            id:'en',
+            name:'English'
+          },
+          // Note:  no './' or '../' prefix, which would root it based on the page's location, not the configured root above.
+          filename: 'us-1.0.js'
```
Comment (jahorton) [inline web/src/samples/complex/root/pages/index.html:58]: After double-checking this again...

https://github.com/keymanapp/keyman/blob/18bb31bbc306717429545f71d7dcfe6eec909f53/web/src/app/web/kmwnative.ts#L78-L91

Looks like I'll need to patch up both this and the other scenario seen in the next `addKeyboards` block below.

File: web/src/samples/build.sh [hunk 2]
```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env bash
+#
+
+# set -x
+set -eu
+
```
Comment (mcdurdin) [inline web/src/samples/build.sh:6]:
```suggestion
```

File: web/src/samples/complex/root/pages/index.html [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+    <!-- Set the viewport width to match phone and tablet device widths -->
+    <meta name="viewport" content="width=device-width,user-scalable=no" />
+
+    <!-- Allow KeymanWeb to be saved to the iPhone home screen -->
+    <meta name="apple-mobile-web-app-capable" content="yes" />
+
+    <!-- Enable IE9 Standards mode -->
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+
```
Comment (mcdurdin) [inline web/src/samples/complex/root/pages/index.html:14]:
```suggestion
```

File: web/src/samples/complex/root/pages/index.html [hunk 4]
```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="content-type" content="text/html; charset=utf-8" />
```
Comment (mcdurdin) [inline web/src/samples/complex/root/pages/index.html:4]:
```suggestion
    <meta charset="utf-8" />
```

**Review by mcdurdin — APPROVED** [review]
Take the meta tag stuff as work todo for all the sample pages as a separate PR

**Pull request merged**